docs/BEST_PRACTICES.md

```diff
@@ -1,4 +1,17 @@
-## Deployment best practices
+## Deployment Guide
```

Comment:
> We need to make sure the documentation is not bounded to a cloud provider( i.e google cloud or AWS) directly.

The current one only targeting for google cloud.

Reply:
I also think the document title as BEST_PRACTICES looks confusing, we could create different documents to explain what is done in this script, maybe referencing the google cloud part differently.

As this document is not referenced in the readme.MD it's not easy to see for people that are willing to use it.

Reply:
It would be much visible to have it on the script it-self. If someone wants to make to ask parameters, it's a subtle change. c3293f785c5f2d7bc23617fb8238a97af0c75594

Reply:
3299fd341c2ab58b350cb60074c73995ab044b9c